packages/editor-ui/src/views/SettingsCommunityNodesView.vue

```diff
@@ -0,0 +1,134 @@
+			<n8n-action-box
+				v-else-if="isQueueModeEnabled"
+				:heading="$locale.baseText('settings.communityNodes.empty.title')"
+				:description="getEmptyStateDescription"
+				:buttonText="$locale.baseText('settings.communityNodes.empty.installPackageLabel')"
+				:calloutText="actionBoxConfig.calloutText"
+				:calloutTheme="actionBoxConfig.calloutTheme"
+				:hideButton="actionBoxConfig.hideButton"
+				@click="openNPMPage"
+			/>
+			<div
+				v-else-if="getInstalledPackages.length === 0"
+				:class="$style.actionBoxContainer"
+			>
+				<n8n-action-box
+					:heading="$locale.baseText('settings.communityNodes.empty.title')"
+					:description="getEmptyStateDescription"
+					:buttonText="$locale.baseText('settings.communityNodes.empty.installPackageLabel')"
+					:calloutText="actionBoxConfig.calloutText"
+					:calloutTheme="actionBoxConfig.calloutTheme"
+					:hideButton="actionBoxConfig.hideButton"
+					@click="openNPMPage"
+				/>
+			</div>
+			<div
+				:class="$style.cardsContainer"
+				v-else
+			>
+				<community-package-card
+					v-for="communityPackage in getInstalledPackages"
+					:key="communityPackage.packageName"
+					:communityPackage="communityPackage"
+				></community-package-card>
+			</div>
+		</div>
+	</SettingsView>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { mapGetters } from 'vuex';
+import SettingsView from './SettingsView.vue';
+import CommunityPackageCard from '../components/CommunityPackageCard.vue';
+
+const PACKAGE_COUNT_THRESHOLD = 31;
+
+export default Vue.extend({
+	name: 'SettingsCommunityNodesView',
+	components: {
+		SettingsView,
+		CommunityPackageCard,
+	},
+	async mounted() {
+		await this.$store.dispatch('communityNodes/fetchInstalledPackages');
+		try {
+			await this.$store.dispatch('communityNodes/fetchAvailableCommunityPackageCount');
+		} finally { }
+	},
+	computed: {
+		...mapGetters('settings', ['isQueueModeEnabled']),
+		...mapGetters('communityNodes', ['getInstalledPackages', 'isLoading']),
+		getEmptyStateDescription() {
+			const packageCount = this.$store.getters['communityNodes/availablePackageCount'];
+			return  packageCount < PACKAGE_COUNT_THRESHOLD ?
+				this.$locale.baseText('settings.communityNodes.empty.description.no-packages') :
+				this.$locale.baseText('settings.communityNodes.empty.description', { interpolate: { count: (Math.floor(packageCount / 10) * 10).toString() } });
+		},
+		actionBoxConfig() {
+			return this.isQueueModeEnabled ? {
+				calloutText: this.$locale.baseText('settings.communityNodes.queueMode.warning'),
+				calloutTheme: 'warning',
+				hideButton: true,
+			} : {
+				calloutText: '',
+				calloutTheme: '',
+				hideButton: false,
+			};
+		},
+	},
+	methods: {
+		openNPMPage() {
+			// TODO: This will open new modal once it is implemented
+			// window.open('https://www.npmjs.com/search?q=keywords:n8n-community-node-package', '_blank');
+			this.$store.dispatch('ui/openModal', 'CommunityPackageInstallModal');
```

Comment:
please make sure modal name is a key in constants like other modals

Reply:
Yep, this the testing piece of code that will be updated when Install dialog is merged.